packages/components/src/Label/Label.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { system } from '@marigold/system';
+import { Text } from '../Text';
+
+// Define custom properties for your component
+type LabelProps = {
+  htmlFor: string;
+};
+
+export const Label = system<LabelProps, 'label'>(
+  ({ variant = 'label', htmlFor, children, ...props }) => {
+    return (
+      <Text
+        as="label"
+        themeSection="form"
+        variant={variant}
+        htmlFor={htmlFor}
```

Comment:
This already is part of `label` no need to add it here.

Reply:
Ok. I wanted to define the prop as a duty attribute. That users get a fault by non-using htmlFor. What do you mean @sebald ?

Reply:
Yeah sorry. That's a good idea!